CHANGES.md

```diff
@@ -4,18 +4,26 @@
 
 ### New
 
-- [dagster-snowflake] Snowflake resource and IO manager now support private key authentication with unencrypted keys
+- Added a new `dagster dev` command that can be used to run both Dagit and the Dagster daemon in the same process during local development. See the new Running Dagster Locally guide in the docs for more information.
+- Added instructions for installing the `dagster` package on M1 and M2 Macs that avoids installation errors when building the `grpcio` package. See [the Installing Dagster guide in the docs](https://docs.dagster.io/getting-started/install) for more information.
+- `create_repository_using_definitions_args` has been added for users to backport their repository definitions to the new `Definitions` API
+- When running Dagit on your local machine, a prompt will now appear that allows you to optionally enter an email address to receive Dagster security updates or subscribe to the Dagster newsletter. This prompt can be dismissed in the UI, or permanently disabled by adding the following to your dagster.yaml file:
+
+```
+nux:
+  enabled: false
+```
+
 - [dagit] Improved query performance on Scheduled Runs page.
 - [dagit] Added or improved document (browser tab) titles throughout Dagit.
+- [dagster-snowflake] Snowflake resource and IO manager now support private key authentication with unencrypted keys
 - [dagster-dbt] The op created when using `load_assets_from_dbt*` is now configurable, allowing you to pass in specific parameters to the underlying dbt command (such as `--full-refresh`). To do so, supply config of the form `{"ops": {"run_dbt_<abcde>": {"config": {"full_refresh": True}}}}` when defining your asset job, or in Dagit.
-- A `dagster dev` which spins up both dagit and the daemon, and requires no `$DAGSTER_HOME` env var to be set
-- `create_repository_using_definitions_args` has been added for users to backport their repository definitions to the new `Definitions` API
 
```

Comment:
### Experimental

- [dagster-dbt] Added a utility to cache compilations from dbt Cloud jobs, allowing software-defined assets to be loaded faster from dbt Cloud jobs.